packages/react-reconciler/src/__tests__/ReactDoubleInvokeEvents-test.internal.js

```diff
@@ -1,747 +0,0 @@
-/**
```

Comment:
Discussed offline - ideally we would preserve the history here, I think you may need to do `git mv` in one commit, then make the changes in the next commit but nbd if it doesn't work out.

Reply:
Okay I had to do some fancy rebasing but I think I split this out. We'll see.